packages/link/src/utils/types.ts

```diff
@@ -131,4 +136,10 @@ export interface FrontOptions {
    * These access tokens are used to initialize crypto transfers flow at 'Select asset step'
    */
   accessTokens?: IntegrationAccessToken[]
+
+  /**
+   * (Optional) The authentication token of the target integration.
+   * Can be used to initialize the crypto transfers flow as an alternative to the target addresses.
+   */
+  transferDestinationToken?: TransferDestinationToken
```

Comment:
I think, this should be the same type as accessTokens, so `IntegrationAccessToken`.